xsd/netex_part_2/part2_journeyTimes/netex_datedVehicleJourney_support.xsd

```diff
@@ -95,6 +95,25 @@
     </xsd:annotation>
     <xsd:restriction base="DatedVehicleJourneyIdType"/>
   </xsd:simpleType>
+    <xsd:element name="NormalDatedVehicleJourneyRef" type="VehicleJourneyRefStructure" substitutionGroup="JourneyRef">
+    <xsd:annotation>
+      <xsd:documentation>Reference to a DATED VEHICLE JOURNEY.</xsd:documentation>
```

Comment:
shouldn't it be NORMAL DATED VEHICLE JOURNEY

Reply:
Corrected (was incomplete copy/paste of annotation/documentation tag), see commit https://github.com/NeTEx-CEN/NeTEx/pull/368/commits/451b61e2a0af010fd8844b43a9423b31c0daa405